README.md

```diff
@@ -1,126 +1,120 @@
-* `-k` gives a path to one or more kubeconfig files to be used by the container to authenticate with the cluster. Paths must be separated by a colon.
-* `-n` gives the network mode of the container. Defaults to `bridge`. See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
+Detailed configuration of the individual specs is explained in [config.md](docs/config.md).
 
-If `-k` is not specified, autodiscovery is performed.
-The autodiscovery first looks for paths in the `$KUBECONFIG` environment variable on the host system, and if the variable is not set or is empty, the default configuration stored in `$HOME/.kube/config` is checked.
+By leveraging resource labels. Automatic configuration will happen by default, that being said, it can be disabled if the environment variable `TNF_DISABLE_CONFIG_AUTODISCOVER` is set
 
-```shell-script
-./run-tnf-container.sh -k ~/.kube/config -t ~/tnf/config -o ~/tnf/output diagnostic generic
+```shell
+TNF_DISABLE_CONFIG_AUTODISCOVER=true
 ```
 
-*Note*: Tests must be specified after all other arguments!
-
-*Note*: The `run-tnf-container.sh` script performs autodiscovery of selected TNF environment variables.  
-Currently supported environment variables include:
-- `TNF_MINIKUBE_ONLY`
-- `TNF_ENABLE_CONFIG_AUTODISCOVER`
-
-#### Running using `docker` instead of `podman`
-
-By default, `run-container.sh` utilizes `podman`.  However, you can configure an alternate container virtualization
-client using `TNF_CONTAINER_CLIENT`.  This is particularly useful for operating systems that do not readily support
-`podman`, such as macOS.  In order to configure the test harness to use `docker`, issue the following prior to
-`run-tnf-container.sh`:
+Pods can be labelled at creation by including the label in their definition, or at any time using the `oc label`
+command. Annotations are almost identical:
 
-```shell script
-export TNF_CONTAINER_CLIENT="docker"
+```shell
+oc label pod test -n tnf test-network-function.com/generic=target
+oc annotate pod test -n tnf test-network-function.com/skip_connectivity_tests=true
 ```
 
-### Building
-
-You can build an image locally by using the command below. Use the value of `TNF_VERSION` to set a branch, a tag, or a hash of a commit that will be installed into the image.
 
-```shell-script
-docker build -t test-network-function:v1.0.5 --build-arg TNF_VERSION=v1.0.5 .
-```
+**NOTE** Currently when this environment variable is set the `generic` and `cnfs` sections of the config file will be
+ENTIRELY replaced with the autodiscovered configuration, to avoid potentially non-obvious errors.
 
-To build an image that installs TNF from an unofficial source (e.g. a fork of the TNF repository), use the `TNF_SRC_URL` build argument to override the URL to a source repository.
+Autodiscovery is currently only supported in the generic spec, where the following applies:
```

Comment:
is this true ? I think all the suites can use autodiscovery

Reply:
I do not think it is true, will remove the sentence

Reply:
Right, I noticed this and forgot to comment. It is actually does all, generic, CNFs, other certification. But we will eliminate this per  suite config concept anyways. So removing it is fine.